src/service/logger/logger.development.ts

```diff
@@ -0,0 +1,119 @@
+import chalk from 'chalk'
+import { EnvUtils, Utils } from '../../util'
+import { LogLevel, Logger } from './logger.types'
+
+interface TatumDevelopmentLoggerOptions {
+  welcome: boolean
+  level: LogLevel
+}
+
+export class TatumDevelopmentLogger implements Logger {
+  private static readonly DISABLE_WELCOME = 'TTM_DISABLE_WELCOME'
+  private static isWelcomeDisabled(): boolean {
+    if (!EnvUtils.isProcessAvailable()) return false
+    return process.env[TatumDevelopmentLogger.DISABLE_WELCOME] === 'true'
+  }
+
+  private static disableWelcome(): void {
+    if (!EnvUtils.isProcessAvailable()) return
+    process.env[TatumDevelopmentLogger.DISABLE_WELCOME] = 'true'
+  }
+
+  private readonly options: TatumDevelopmentLoggerOptions
+  private readonly logger = console
+
+  private readonly _trace = chalk.gray
+  private readonly _debug = chalk.gray
+  private readonly _info = chalk.white
+  private readonly _warn = chalk.bold.yellow
+  private readonly _error = chalk.bold.red
+
+  private readonly _DEBUG = chalk.bgGray.white(' DEBUG ')
+  private readonly _INFO = chalk.bgCyan.gray(' INFO ')
+  private readonly _WARN = chalk.bgYellowBright.black(' WARN ')
+  private readonly _ERROR = chalk.bgRedBright.bold.yellow(' ERROR ')
+
+  constructor(options: Partial<TatumDevelopmentLoggerOptions> = {}) {
+    const defaultOptions: TatumDevelopmentLoggerOptions = {
+      welcome: EnvUtils.isDevelopment() && !TatumDevelopmentLogger.isWelcomeDisabled(),
+      level: LogLevel.INFO,
+    }
+
+    this.options = { ...defaultOptions, ...options }
+
+    if (this.options.welcome) {
+      this.welcome()
+      TatumDevelopmentLogger.disableWelcome()
+    }
+  }
+
+  private welcome(): void {
+    this.logger.log([LOGO, Utils.randomElementOf(WELCOME_MESSAGES)].join('\n'))
+  }
+
+  trace(...args: unknown[]): void {
+    if (this.options.level > LogLevel.TRACE) return
+    this.logger.trace(this._trace(...args))
+  }
+
+  debug(...args: unknown[]): void {
+    if (this.options.level > LogLevel.DEBUG) return
+    this.logger.debug(this._DEBUG, this._debug(...args))
+  }
+
+  info(...args: unknown[]): void {
+    if (this.options.level > LogLevel.INFO) return
+    this.logger.info(this._INFO, this._info(...args))
+  }
+
+  warn(...args: unknown[]): void {
+    if (this.options.level > LogLevel.WARN) return
+    this.logger.warn(this._WARN, this._warn(...args))
+  }
+
+  error(...args: unknown[]): void {
+    if (this.options.level > LogLevel.ERROR) return
+    this.logger.error(this._ERROR, this._error(...args))
+  }
+}
+
+const LOGO = `--------------------------------------------------------------------------------
```

Comment:
As I mentioned to you, I personally preferred the old logo more, but if both you and Shiv believe this one to be better, you can of course keep this one.